loss-detection-app/pipelines/object_detection/2/pipeline.json

```diff
@@ -1,14 +1,12 @@
 {
-    "name": "object_detection",
-    "version": 1,
     "type": "GStreamer",
-    "template":"urisourcebin uri=\"{source[uri]}\" ! queue ! decodebin ! videoconvert name=\"videoconvert\" ! video/x-raw ! queue leaky=upstream ! gvadetect inference-id=inf0 model=\"{models[object_detection][1][network]}\" model-proc=\"{models[object_detection][1][proc]}\" name=\"detection\" ! gvametaconvert converter=json method=detection source=\"{source[uri]}\" name=\"jsonmetaconvert\" include-no-detections=true ! queue ! gvametapublish method=file filepath=\"/results/results.txt\" ! queue ! appsink name=appsink",
+    "template":"urisourcebin uri=\"{source[uri]}\" ! queue ! decodebin ! videoconvert name=\"videoconvert\" ! video/x-raw ! queue leaky=upstream ! gvadetect model=\"{models[object_detection][1][network]}\" model-proc=\"{models[object_detection][1][proc]}\" name=\"detection\" ! gvametaconvert name=\"metaconvert\" source=\"{source[uri]}\" add-empty-results=true ! queue ! gvametapublish name=\"destination\" file-path=\"/results/results.txt\" ! queue ! appsink name=appsink",
```

Comment:
If the only different here is the explicit file-path we can remove this pipeline version altogether. We support sending file path as an argument to metapublish

Reply:
Removed, will document as part of destination request